### PR TITLE
Budget

### PR DESCRIPTION
**Pull request by mau-fyi**
Adds a budget servlet that returns a list of html lines to integrate.
They appear as <p>Trip budget: __, Expected contribution: ___</p> or <p>error msg</p>

### CODE REVIEW

File: src/main/java/com/google/sps/app/servlets/GetTrips.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.google.sps.app.servlets;
+    package com.google.sps.app.servlets;
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/GetTrips.java]: nit: remove the extra indentation

File: src/main/java/com/google/sps/app/servlets/getBudget.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+     * @param req contains a userID
+     * @param resp returns JSON  with list of budgets and expected contributions
+     * @throws ServletException
+     * @throws IOException
+     */
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String userID = req.getHeader("userID");
+
+        List<String> associatedTripIDs = getAssociatedTrips(userID);
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/getBudget.java:39]: Can you rename this to getTripParticipants, since that is what it is doing

**Review by sbahaud — APPROVED** [review]
Approving with a comment

File: src/main/java/com/google/sps/app/servlets/GetTrips.java (hunk 1 above)
Comment (mau-fyi) [reply src/main/java/com/google/sps/app/servlets/GetTrips.java]: Oddly enough this isn't in my local repository so i can't change it there.

File: src/main/java/com/google/sps/app/servlets/getBudget.java (hunk 2 above)
Comment (mau-fyi) [reply src/main/java/com/google/sps/app/servlets/getBudget.java:39]: It's actually supposed to get trips IDs. I copied the query from another method so I'll fix it  now.

**Pull request merged**